site/content/en/docs/tasks/manage/setup_multikueue.md

```diff
@@ -60,12 +60,19 @@ kubectl config use-context manager-cluster
 ```
 {{% /alert %}}
 
-### Jobset CRD only install
+### JobSet installation
 
-As mentioned in the [MultiKueue Overview](/docs/concepts/multikueue/#jobset) section, in the manager cluster only the JobSet CRDs should be installed, you can do this by running:
+If you are using Kueue in version 0.7.0 or newer install the JobSet on the
+management cluster (see [JobSet Installation](https://jobset.sigs.k8s.io/docs/installation/)
+for more details). We recommend using JobSet 0.5.1+ for MultiKueue.
```

Comment:
```suggestion
for more details). We recommend using JobSet 0.5.1 or newer for MultiKueue.
```

I'm fine putting it as a requirement, rather a recommendation.

Reply:
I update the "or newer". Regarding the requirement or recommendation. I rephrased to just say please install. WDYT?